akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/StashBuffer.scala

```diff
@@ -4,12 +4,12 @@
 
 package akka.actor.typed.javadsl
 
-import java.util.function.Consumer
 import java.util.function.{ Function => JFunction }
```

Comment:
used in `wrap`, don't we want japi there?

Reply:
Wrap we want to keep non-japi because it is expected to be a pure function.